client/src/containers/Profile.js

```diff
@@ -1,36 +1,109 @@
 import React, { PropTypes, Component } from 'react';
+import moment from 'moment';
 import { connect } from 'react-redux';
+import { pushState } from 'redux-router';
 import { getUserInfo } from '../actions/user.js';
+import { fetchCodesByUserId } from '../actions/code.js';
 
+// TODO make repsonsive design
+// TODO figure out how to get all code smells that this user has reviewed
 class Profile extends Component {
 
     constructor(props) {
         super(props);
+        this.routeToCodeReview = this.routeToCodeReview.bind(this);
+        this.state = {
+            receivedCodes: false,
```

Comment:
We want to have the least amount of state in handled by a component. For things like input fields and stuff that's okay, but holding a flag to check whether or not code has been received falls on the other side.